cheat.php

```diff
@@ -713,6 +776,46 @@ function ExecuteRequest( $Method, $URL, $Data = [] )
 	return $Data;
 }
 
+function GetLocalScriptHash( )
+{
+	return sha1_file( __FILE__ );
+}
+
+function GetRepositoryScriptHash( )
+{
+	$c_r = curl_init( );
+
+	curl_setopt( $c_r, CURLOPT_URL, 'https://api.github.com/repos/SteamDatabase/SalienCheat/git/trees/master' );
+	curl_setopt_array( $c_r, [
+		CURLOPT_USERAGENT      => 'SalienCheat (https://github.com/SteamDatabase/SalienCheat/)',
+		CURLOPT_RETURNTRANSFER => true,
+		CURLOPT_ENCODING       => 'gzip',
+		CURLOPT_TIMEOUT        => 5,
+		CURLOPT_CONNECTTIMEOUT => 5,
+		CURLOPT_HEADER         => 1,
```

Comment:
Since we don't need the header, you can remove this line, and the 2 lines below:
```
$HeaderSize = curl_getinfo( $c_r, CURLINFO_HEADER_SIZE );
$Data = substr( $Data, $HeaderSize );
```